frontend/src/store/modules/place/mutations.js

```diff
@@ -41,5 +41,23 @@ export default {
 
     SET_CURRENT_PLACE_REVIEW_DISLIKE_COUNT: (state, { reviewId, count }) => {
         state.currentPlaceReviews.byId[reviewId].dislikes = count;
+    },
+
+    SET_PLACE_REVIEW_LIKE_STATE: (state, { placeId, likeState }) => {
+        state.places.find( (place) => { 
```

Comment:
create findPlaceByReview(reviewId) function and move logic inside
```
const place = state.places.find( (place) => { 
      return place.review.id === parseInt(placeId); 
});

return place;
```
mutation would look like `SET_PLACE_REVIEW_LIKE_STATE`
```
SET_PLACE_REVIEW_LIKE_STATE((state, { reviewId, likeState })) {
   const place = findPlaceByReview(state.places, reviewId);
   place.review.like = likeState;
}
```